packages/turbo/ios/RNVisitableViewModule.swift

```diff
@@ -17,6 +17,18 @@ class RNSessionModule: RCTEventEmitter {
   
   @objc
   private var defaultSession = RNSession()
+    
+  @objc
+  public func setConfiguration(
+    _ sessionHandle: NSString,
+    resolver resolve: @escaping RCTPromiseResolveBlock,
+    rejecter reject: @escaping RCTPromiseRejectBlock) -> Void {
```

Comment:
@pklatka currently we are not using `reject`, right? Should we add it later at some point? Or we don't need custom logic here?

Reply:
Hmm, we might use reject when we add something to RNSession that could potentially block us from using, for example, a webview, but for now, we don't need it.